gumrya-web/src/main/java/gumrya/service/UserService.java

```diff
@@ -2,22 +2,59 @@
 
 import gumrya.dto.UserRequest;
 import gumrya.dto.UserResponse;
+import gumrya.exception.DuplicateException;
+import gumrya.exception.NotFoundException;
+import gumrya.user.domain.model.User;
+import gumrya.user.domain.repository.UserRepository;
+import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
 
 @Service
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
 public class UserService {
+
+    private final UserRepository userRepository;
+
+    @Transactional
     public UserResponse save(UserRequest userRequest) {
-        return null;
+        if (existsByEmail(userRequest.getEmail())) {
+            throw new DuplicateException("등록하려는 이메일의 사용자가 이미 존재합니다");
+        }
+        User user = new User(userRequest.getEmail(), userRequest.getPassword(), userRequest.getNickname(),
+                userRequest.getIntroduction(), userRequest.getProfileUrl(), userRequest.getGithubUrl());
+        User saveUser = userRepository.save(user);
+        return new UserResponse(saveUser);
```

Comment:
```suggestion
        User user = new User(userRequest.getEmail(), userRequest.getPassword(), userRequest.getNickname(),
                userRequest.getIntroduction(), userRequest.getProfileUrl(), userRequest.getGithubUrl());
                
        User saveUser = userRepository.save(user);
        return new UserResponse(saveUser);
```
흐름이 변경되는곳에 줄을 띄워줘도 괜찮을거같아요